src/app/public/auth/auth.component.ts

```diff
@@ -48,6 +48,10 @@ export class LoginComponent implements CognitoCallback, LoggedInCallback, OnInit
         if (message != null) { //error
             this.errorMessage = message;
             console.log("result: " + this.errorMessage);
+            if (this.errorMessage === 'User is not confirmed.') {
```

Comment:
I'll merge this for now, but this is not the best check. It's definitely not going to hurt, but the actual message might change and this conditional statement will default to false.